src/components/GsocTsoc/GsocTsoc.jsx

```diff
@@ -73,41 +73,43 @@ const GsocTsoc = () => {
     <div>
       <div>
         <h4 className="text-center">
-          About Google summer of code & Terasology summer of code{" "}
+          {`About Google summer of code & Terasology summer of code${" "}`}
         </h4>
         <div className="container my-4">
-          <div className="home-underline"></div>
+          <div className="home-underline" />
 
           <Row className="justify-content-center">
             <div className="col-md-10">
               <div className="text-center  gsoc_tsoc_content">
-                <b>Google Summer of Code (GSoC)</b> is a global program focused
-                on bringing more student developers into open source software
-                development. Every year more than 1200 students are accepted.
-                Accepted students work with a mentor and become a part of the
-                open source community. Many become lifetime open source
-                developers. Spend your summer break writing code and learning
-                about open source development while earning a stipend. For more
-                information visit{" "}
+                <b>Google Summer of Code (GSoC) </b>
+                is a global program focused on bringing more student developers
+                into open source software development. Every year more than 1200
+                students are accepted. Accepted students work with a mentor and
+                become a part of the open source community. Many become lifetime
+                open source developers. Spend your summer break writing code and
+                learning about open source development while earning a stipend.
+                For more information visit{" "}
```

Comment:
This new-line rule is kinda dumb when applied to actual prose text with some highlights :see_no_evil: